math/mean.go

```diff
@@ -0,0 +1,31 @@
+package maths
+
+import (
+	"golang.org/x/exp/constraints"
+)
+
+type Digit interface {
+	constraints.Float | constraints.Integer
+}
+
+func Mean[T Digit](values []T) float64 {
+
+	// Validating the empty array
```

Comment:
No need for these kind of comments. They usually don't provide value.

When adding comments to code, there are usually two things that you should consider:

1. are you doing something that is very complex that will not be understood by someone reading your code.
2. are you explain why you're doing something rather than what you're doing.

If none of the above cases apply then it is (usually) just considered noise.